src/SWP/Bundle/MultiTenancyBundle/DependencyInjection/SWPMultiTenancyExtension.php

```diff
@@ -71,6 +72,7 @@ public function loadPhpcr($config, YamlFileLoader $loader, ContainerBuilder $con
         }
 
         array_push($config['route_basepaths'], $config['content_basepath']);
```

Comment:
why not `array_push($config['route_basepaths'], $config['content_basepath'], $config['menu_basepath']);`?